README.md

```diff
@@ -227,10 +227,21 @@ $ npm run component-gen
 
 ## ➤ How to release
 
-As mentioned before, we are using [lerna](https://github.com/lerna/lerna), to be able to publish you need enable your [two-factor authentication in npm](https://docs.npmjs.com/configuring-two-factor-authentication)
```

Comment:
Why you removed the NPM OTP?

Reply:
They fixed right now. We don't need it anymore